azurerm/internal/services/logic/logic_app_trigger_recurrence_resource.go

```diff
@@ -65,6 +65,12 @@ func resourceArmLogicAppTriggerRecurrence() *schema.Resource {
 				Optional:     true,
 				ValidateFunc: validation.IsRFC3339Time,
 			},
+
+			"time_zone": {
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
the update test should confirm this, but I assume this defaults to UTC on the server-side?

Reply:
The documentation seems to indicate this - I've gone and set the default `time_zone` value to UTC and updated an old test which set just the `start_time` to verify that `UTC` is set as the default.